Correcting the Installation Documentation for Windows users (#298)
hexatomic/hexatomic: docs/dev/src/development/setup.md

We suggest that you use **OpenJDK** since more recent versions of the *Oracle JDK* have a more restricted license.
OpenJDK is included in the package management repositories (dpkg (via apt), RPM, etc.) of most Linux distributions.
For stand-alone packages, you can use the installers distributed by the [AdoptOpenJDK project](https://adoptopenjdk.net).
AdoptOpenJDK packages are are available for Windows, MacOS, and Linux. If you are asked to choose a JVM, both HotSpot and OpenJ9 work with Hexatomic.

> When installing the AdoptOpenJDK on Windows, make sure to specify that the JAVA_HOME environment variable is set.

Reviewer: ```suggestion
When installing the AdoptOpenJDK on Windows, make sure that the option to set the `JAVA_HOME` environment variable is activated.
Otherwise you will have to set the environment variable manually.
```

I assume that this is a checkbox or similar that needs to be checked during the installation, otherwise the env var isn't set?

- Open the *Install Wizard* via the menu **Help > Install New Software...**
- In the **Work with:** field, enter the URL of the Update Site for the plugin, given in brackets below.

> - Select the plugins you want to install, and install it with the help of the wizard.

Reviewer: ```suggestion
- Select the plugins you want to install, and install them with the help of the wizard.
```
Unrelated to the PR: fix typo.

> - (Installing IDE plugins might trigger security warnings on Windows)

Reviewer: ```suggestion
- Installing IDE plugins sometimes triggers security warnings, because the code in the plugins has not been [signed](https://en.wikipedia.org/wiki/Code_signing). Often, code authors will simply have avoided the cost of purchasing a certificate to do the signing with, and in these cases the warnings are false alarms. However, we encourage you to carefully check the plugins you install, just as you would when you install other software.
```

This is correct, although I think it'd be helpful to have a slightly more detailed explanation (also, you will get security warnings on other OSs as well). I have suggested something along these lines, unless I have got it wrong and on Windows, installing IDE plugins triggers the Windows Defender or whatever it is called?
